packages/hooks/src/createAddHook.js

```diff
@@ -68,6 +69,8 @@ function createAddHook( hooks ) {
 				runs: 0,
 			};
 		}
+
+		doAction( 'hookAdded', hookName, namespace, callback, priority );
```

Comment:
Should we guard against `'hookName' === 'hookAdded'`? :)

Reply:
that is what i was wondering, but i think fine to keep it simple for now

Reply:
Once I added `jest.fn` to spy on it, it revealed that calling:
```js
addAction( 'hookAdded', 'my_callback', hook_added_spy );
```
triggers `hookAdded` action. I'll add condition to prevent that.